examples/with-urql-graphql-codegen/src/pages/character/[id].tsx

```diff
@@ -0,0 +1,32 @@
+import { GetServerSideProps } from 'next';
+import { getClient } from '../../lib/urql';
+import {
+  CharacterDocument,
+  CharacterQueryVariables
+} from '../../generated/graphql';
+
+import Character from '../../components/character';
+
+const CharacterPage: React.FC = () => {
+  return (
+    <>
+      <h1>Character</h1>
+      <Character />
+    </>
+  );
+};
+
+export const getServerSideProps: GetServerSideProps = async ({ params }) => {
+  const client = getClient();
+  await client
+    .query<any, CharacterQueryVariables>(CharacterDocument, {
+      id: params.id as string
+    })
+    .toPromise();
+
+  return {
+    props: {}
+  };
```

Comment:
The data here won't be properly populated in the page, you need to return an initial state here or this will have issues with the other data fetching methods and in client-side navigations.